lib/pejs.ts

```diff
@@ -23,11 +23,38 @@ export class PEJS {
    * @return the evaluation results specify what was expected and was fulfilled and also specifies what requirement described in the input descriptor
    * was not fulfilled by the presentation.
    */
-  public evaluate(presentationDefinition: PresentationDefinition, presentation: Presentation): EvaluationResults {
```

Comment:
Copy doc from the ticket.
https://sphereon.atlassian.net/browse/ESSIFI-155

Reply:
I don't think there is a list of dids on the verifier side. The did from the presentation is used.